src/util/Options.js

```diff
@@ -69,6 +69,10 @@
  * [User Agent](https://discord.com/developers/docs/reference#user-agent) header
  * @property {PresenceData} [presence={}] Presence data to use upon login
  * @property {IntentsResolvable} intents Intents to enable for this connection
+ * @property {number} [fetchGuildTimeout=15000] Time in milliseconds that bots with the `GUILDS` intent should hang
+ * and wait for missing guilds to be received before starting the bot
+ * If not specified, the default is 15 seconds.
+ * (0 to skip this timeout)
```

Comment:
A few non-binding rephrasing suggestions:
- `hang and wait`: seems redundant
- `missing`: feels a bit misleading (might just be me)
- `starting the bot`: `Client#login` starts it. It's just not ready yet, a small nit.
- `skip this timeout`: `not wait` seems to be a bit more concise

```suggestion
 * @property {number} [waitGuildTimeout=15000] Time in milliseconds that bots with the `GUILDS` intent should wait for
 * initial guild's data to be received before emitting ready. (0 to not wait)
```